public/js/components/App.js

```diff
@@ -29,7 +29,7 @@ const App = React.createClass({
 
   getInitialState() {
     return {
-      searchOn: false
+      searchOn: true
```

Comment:
It opens up UX questions: does that mean it auto-focuses on open? That means key shortcuts probably don't work, and the sources tree should probably be the thing we focus first. Beginners would probably be more confused sitting at a blank input rather than the sources tree.